conftest.py

```diff
@@ -150,21 +150,29 @@ def sr_disk_wiped(host, sr_disk):
     host.ssh(['wipefs', '-a', '/dev/' + sr_disk])
     yield sr_disk
 
+def get_default_ref(request):
```

Comment:
Maybe `get_default_vm_ref` to be more explicit instead?
`get_default_vm_key` is probably even better because you use it in `def vm_key(request)`